lib/server.go

```diff
@@ -107,7 +117,7 @@ func parseCmd(line string) (cmd string, args []string, err error) {
   }
 
   args = strings.Split(line, " ")
-  if len(args) < 2 {
+  if (args[0] != "reset") && args[0] != "exit" && len(args) < 2 {
```

Comment:
1. can remove the parens.
2. should check to be sure `args` has elements, otherwise `args[0]` will cause a run-time panic.

```go
if len(args) < 2 && len(args) > 0 && args[0] != "reset" && args[0] != "exit" {
...
```

I'm also not sure how this function is being called, but usually the format of command line arguments in golang follow the pattern `["program name", "arg1", "arg2", ...]`. That means that `arg[0]` will just be the program name. You might want to test that you don't really want to check `arg[1]`, i.e.  `args[1] != "reset" && args[1] != "exit"`.

See [this explanation](https://gobyexample.com/command-line-arguments).